src/harness/harness.ts

```diff
@@ -938,6 +938,9 @@ namespace Harness {
                                 }
                         }
                     }
+                    else if (name === "suppressOutputPathCheck") {

```

Comment:
it already handles a set of additional extensions, and they are all declarative.